assets/css/style.css

```diff
@@ -1,276 +1,428 @@
-  margin-top: 0.5rem;
-  list-style: none;
+.brands {
+  margin: 15px 0 15px -5px;
   display: flex;
+  list-style: none;
+}
+
+.brands i {
+  padding-left: 10px;
+  color: #505f79;
+  font-size: 20px;
+  font-style: normal;
+}
+
+.button-mobile {
+  margin-right: 15px;
+}
+
+.desktop-img {
+  display: none;
 }
 
-.headline-icon {
-  padding: 0.7rem;
+.mobile-img {
+  display: block;
 }
 
-.card {
+.works-portfolio {
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  background: var(--white);
-  border-radius: 20px;
-  margin-bottom: 10rem;
-  padding-bottom: 2rem;
-  width: 80vw;
+  justify-content: center;
 }
 
-.card-image {
+.work-inside {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
   width: 90%;
-  margin-top: 1.5rem;
+  list-style: none;
+  padding: 8rem 1.6rem;
+  row-gap: 5em;
+  justify-content: center;
+  margin: 0 auto;
 }
 
-.card-body {
+.project {
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
-  margin: 2rem;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 20px;
+  background-color: #fff;
 }
 
-.card-body h2 {
+.work-title {
+  font-family: var(--font-family-style);
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 44px;
   color: #172b4d;
-  color: var(--color-headings);
-  font-size: 3.2rem;
 }
 
-.card-specs {
+.text-work {
+  /* Auto layout */
+  margin: 0 auto;
+  padding: 8rem 1.6rem;
+  row-gap: 5em;
+  justify-content: center;
+}
+
+.work-details {
   display: flex;
+  column-gap: 0.75em;
   align-items: center;
-  column-gap: 1rem;
-  margin-top: 1rem;
 }
 
-.text {
-  color: var(--text-color);
-  font-size: 1.4rem;
-  font-weight: 600;
+.p-details {
+  font-size: 13px;
+  font-weight: 800;
+  line-height: 16px;
+  color: #344563;
 }
 
-.text-light {
+.ul-details {
+  display: flex;
+  align-content: center;
+  padding-left: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
   color: #7a869a;
-  font-size: 1.4rem;
+  column-gap: 2em;
+  list-style: initial;
 }
 
-.card-body p {
-  margin: 0.5rem 1rem 0 0;
-  font-family: var(--font-family-style);
-  font-style: normal;
+.ul-details > li {
+  list-style: disc;
+}
+
+.p-ul-details {
+  font-size: 15px;
   font-weight: 400;
-  font-size: 13px;
   line-height: 24px;
-  color: var(--color-paragraphs);
+  color: #344563;
 }
 
-.categories {
+.ul-tags {
   display: flex;
-  align-items: center;
-  gap: 0.8rem;
   list-style: none;
-  margin-top: 1rem;
+  column-gap: 4px;
+  margin: 15px 0 15px 0;
 }
 
-.categories li {
-  font-size: 1.1rem;
-  background: #ebebff;
-  color: var(--accent-color);
-  padding: 0.5rem 1rem;
+.ul-tags li {
+  padding: 3px 14px 3px 14px;
+  background-color: #ebebff;
   border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  font-weight: 500;
 }
 
-.btn {
-  padding: 1.5rem 2.2rem;
-  margin-top: 3rem;
-  border-radius: 10px;
-  border: 1px solid #396df2;
+.btn-in {
```

Comment:
- [ ] [optional] The elements of the form should be centered on the page including the ``` Get in touch``` button, please fix it

-Your design:
![Screenshot_20220506_174115](https://user-images.githubusercontent.com/25227022/167185887-a6e6d3ad-865f-4b8d-8835-96dfa90d9fb5.png)

-Figma design:
![Screenshot_20220506_174314](https://user-images.githubusercontent.com/25227022/167186287-e7d1fb98-4075-4523-8db2-0dab74be952f.png)